chef/cookbooks/ceph/recipes/osd.rb

```diff
@@ -68,7 +68,8 @@
 
   if is_crowbar?
     node.set["ceph"]["osd_devices"] = [] if node["ceph"]["osd_devices"].nil?
-    unclaimed_disks = BarclampLibrary::Barclamp::Inventory::Disk.unclaimed(node).sort
+    min_size_blocks = node["ceph"]["osd"]["min_size_gb"] * 1024 * 1024 * 2
+    unclaimed_disks = BarclampLibrary::Barclamp::Inventory::Disk.unclaimed(node).sort.select {|d| d.size >= min_size_blocks}
```

Comment:
Ah, now I get what you're on about :-)

The size there ultimately comes from `/sys/block/<dev>/size`. My understanding is that this always presents a figure in 512B blocks, regardless of the physical block size.